kwok/charts/crds/karpenter.sh_nodeclaims.yaml

```diff
@@ -1 +1,459 @@
-../../../pkg/apis/crds/karpenter.sh_nodeclaims.yaml
\ No newline at end of file
+---
```

Comment:
Why are you removing the symlink?

Reply:
I wanted to copy the yamls over so we could add our own validation. Am I missing something by removing these?